plugin/src/main/java/org/godotengine/openxr/vendors/GodotPanelApp.kt

```diff
@@ -60,9 +61,11 @@ class GodotPanelApp : GodotActivity() {
 		super.onCreate(savedInstanceState)
 	}
 
-	override fun getCommandLine(): MutableList<String> {
+	@CallSuper
+	protected override fun updateCommandLineParams(args: Array<String>) {
 		// Force XR to be turned off.
-		return mutableListOf("--xr_mode_regular", "--xr-mode", "off")
+		val args = args + arrayOf("--xr_mode_regular", "--xr-mode", "off")
+		super.updateCommandLineParams(args);
```

Comment:
We should be able to revert this once https://github.com/godotengine/godot/pull/102669 is submitted.